pynicotine/plugins/core_commands/__init__.py

```diff
@@ -120,7 +133,92 @@ def __init__(self, *args, **kwargs):
                 "usage": ["[room]"],
                 "aliases": ["l"],
                 "group": _("Chat Rooms")
-            }
+            },
+            "add": {
```

Comment:
Does adding these commands to the `commands` dict, and overriding them in the `private_chat_commands` dict work?

Reply:
Yes, it does. Infact that is the way I wanted to approach it at first to avoid so much duplication in the definitions.

However this results in the about of the /help command being sorted wrong. I tried sorted() but the alphabetical sort order of the group names doesn't make logical sence. I suppose if we manually moved the "Commands" group to the top, then it would be okay.

Reply:
Done the overriding dict structure as suggested. Unfortunately it is necessary to re-define all the attribute data for each duplicated command item.